test_requirements.txt

```diff
@@ -9,7 +9,7 @@ pytest-timeout==2.0.1
 types-requests==2.25.6
 conda-merge==0.1.5
 gitpython==3.1.7
-lightning-bolts==0.5.0
+lightning-bolts==0.4.0
```

Comment:
how do we ensure environment.yml and this test_requirements file don't go out of sync?

Reply:
Good point - I have updated the environment.yml to read from this file